test/1_test_fullpath.js

```diff
@@ -184,37 +184,43 @@ contract('Voucher tests', async (addresses) => {
 
       //Check BosonRouter state
       assert.equal(
-        await contractBosonRouter.correlationIds(users.seller.address),
+        await contractBosonRouter.getCorrelationId(users.seller.address),
         1,
         'Correlation Id incorrect'
       );
 
       //Check VocherKernel State
-      const promise = await contractVoucherKernel.promises(promiseId1);
-      assert.equal(promise.promiseId, promiseId1, 'Promise Id incorrect');
-      assert.isTrue(promise.nonce.eq(constants.ONE), 'Nonce is incorrect');
+      const promiseData = await contractVoucherKernel.getPromiseData(
+        promiseId1
+      );
+      assert.equal(promiseData[0], promiseId1, 'Promise Id incorrect');
+      assert.isTrue(promiseData[1].eq(constants.ONE), 'Nonce is incorrect');
+      assert.isTrue(promiseData[2].eq(new BN(constants.PROMISE_VALID_FROM)));
+      assert.isTrue(promiseData[3].eq(new BN(constants.PROMISE_VALID_TO)));
+      assert.isTrue(promiseData[4].eq(constants.ZERO));
+
+      const promiseSeller = await contractVoucherKernel.getSupplyHolder(
+        tokenSupplyKey1
+      );
+
       assert.strictEqual(
-        promise.seller,
+        promiseSeller,
         users.seller.address,
         'Seller incorrect'
       );
-      assert.isTrue(promise.validFrom.eq(new BN(constants.PROMISE_VALID_FROM)));
-      assert.isTrue(promise.validTo.eq(new BN(constants.PROMISE_VALID_TO)));
-      assert.isTrue(promise.price.eq(new BN(constants.PROMISE_PRICE1)));
-      assert.isTrue(promise.depositSe.eq(new BN(constants.PROMISE_DEPOSITSE1)));
-      assert.isTrue(promise.depositBu.eq(new BN(constants.PROMISE_DEPOSITBU1)));
-      assert.isTrue(promise.idx.eq(constants.ZERO));
 
-      const orderPromiseId = await contractVoucherKernel.ordersPromise(
+      const promiseOrderData = await contractVoucherKernel.getOrderCosts(
         tokenSupplyKey1
       );
-      assert.strictEqual(
-        orderPromiseId,
-        promiseId1,
-        'Order Promise Id incorrect'
+      assert.isTrue(promiseOrderData[0].eq(new BN(constants.PROMISE_PRICE1)));
```

Comment:
Please use constants instead of hard-coded indexes for promiseOrderData (same as remark above)

Reply:
Remaining hard-coded indexes in lines 232,234,237